components/NFTMint/WriterMessage.vue

```diff
@@ -57,49 +57,67 @@
-          <IconGift />
-        </template>
-      </Label>
+    <template v-if="shouldShowSettings">
       <div
-        class="flex justify-center gap-[12px] text-dark-gray text-[14px] items-center bg-[#E6F4F2] rounded-[4px] py-[4px]"
+        class="flex flex-col justify-start gap-[32px] pt-[20px] pb-[24px] px-[40px] border-2 border-[#E6F4F2] rounded-[16px] w-full"
       >
-        <span>{{ $t('NFTPortal.label.reserve.input') }}</span>
-        <input
-          ref="nameInput"
-          :placeholder="placeholder"
-          type="number"
-          :max="premintAmount"
-          min="0"
-          class="w-[65px] bg-transparent border-0 border-b-2 border-black outline-none text-center placeholder-medium-gray focus:outline-none"
-          @change="(value) => $emit('update-reserve', value)"
-        />
+        <Label
+          class="w-min text-like-green"
+          :text="$t('NFTPortal.label.message.reserve')"
+          tag="div"
+          preset="p6"
+          valign="middle"
+          align="left"
+          content-class="whitespace-nowrap"
+        >
+          <template #prepend>
+            <IconGift />
+          </template>
+        </Label>
+        <div
+          class="flex justify-center gap-[12px] text-dark-gray text-[14px] items-center bg-[#E6F4F2] rounded-[4px] py-[4px]"
+        >
+          <span>{{ $t('NFTPortal.label.reserve.input') }}</span>
+          <input
+            ref="nameInput"
+            :placeholder="placeholder"
+            type="number"
+            :max="premintAmount"
+            min="0"
+            class="w-[65px] bg-transparent border-0 border-b-2 border-black outline-none text-center placeholder-medium-gray focus:outline-none"
+            @change="(value) => $emit('update-reserve', value)"
+          />
+        </div>
       </div>
+
       <div
-        v-if="shouldShowInitialBatchSettings"
-        class="flex justify-center gap-[12px] text-dark-gray text-[14px] items-center bg-[#E6F4F2] rounded-[4px] py-[4px]"
+        class="flex flex-col justify-start gap-[32px] pt-[20px] pb-[24px] px-[40px] border-2 border-[#E6F4F2] rounded-[16px] w-full"
       >
-        <span>{{ $t('NFTPortal.label.initialBatch.input') }}</span>
-        <select ref="batchInput" @change="(value) => $emit('update-initial-batch', value)">
-          <option v-for="_, i in Array(maxInitialBatch)" :key="i" :value="i">
-            {{ basePrice * Math.pow(2, i)}}
-          </option>
-        </select>
+        <Label
+          class="w-min text-like-green"
+          :text="$t('NFTPortal.label.message.mintSettings')"
+          tag="div"
+          preset="p6"
+          valign="middle"
+          align="left"
+          content-class="whitespace-nowrap"
+        >
+          <template #prepend>
+            <IconMint />
+          </template>
+        </Label>
+        <div
+          class="flex justify-center gap-[12px] text-dark-gray text-[14px] items-center bg-[#E6F4F2] rounded-[4px] py-[4px]"
```

Comment:
For the updated UI, I suggest to remove the cyan-ish bg to make the layout cleaner
```suggestion
          class="flex justify-center gap-[12px] text-dark-gray text-[14px] items-center"
```